config/initializers/devise_token_auth.rb

```diff
@@ -2,7 +2,12 @@
 
 DeviseTokenAuth.setup do |config|
   bypass_sign_in = false 
-  config.default_confirm_success_url = "confirmed"
+  
+  #config.default_confirm_success_url = "http://localhost:3000"
+
+  # By default, url points to testing. For production override param "redirect_url" with
+  # production webhost
+  config.default_password_reset_url = "http://localhost:3000/api/auth/users/password/edit"
```

Comment:
Avoid hardcoded host, if not when it is deployed in production would not work.

```suggestion
  config.default_password_reset_url = "/api/auth/users/password/edit"
```